pkg/cli/cmd/radinit/aws.go

```diff
@@ -71,21 +84,36 @@ func (r *Runner) enterAWSCloudProvider(ctx context.Context) (*aws.Provider, erro
 	}, nil
 }
 
-func (r *Runner) getAccountId(ctx context.Context, accessKeyID, secretAccessKey string) (string, error) {
-	callerIdentityOutput, err := r.awsClient.GetCallerIdentity(ctx, QueryRegion, accessKeyID, secretAccessKey)
+func (r *Runner) getAccountId(ctx context.Context) (string, error) {
+	callerIdentityOutput, err := r.awsClient.GetCallerIdentity(ctx)
 	if err != nil {
-		return "", clierrors.MessageWithCause(err, "AWS credential verification failed.")
+		return "", clierrors.MessageWithCause(err, "AWS Cloud Provider setup failed, please use aws configure to set up the configuration. More information :https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html")
 	}
 
 	if callerIdentityOutput.Account == nil {
 		return "", clierrors.MessageWithCause(err, "AWS credential verification failed: Account ID is nil.")
 	}
 
-	return *callerIdentityOutput.Account, nil
+	accountID := *callerIdentityOutput.Account
+	addAccountID, err := prompt.YesOrNoPrompt(fmt.Sprintf(confirmAWSAccountIDPromptFmt, accountID), prompt.ConfirmYes, r.Prompter)
+	if err != nil {
+		return "", err
+	}
+
+	if !addAccountID {
```

Comment:
Shouldn't this be true so that the user can enter account id? Like the logic here is that if `addAccountID` is false then ask the user to enter AWS account id? Am I missing something here?

Reply:
we retrieve the account id using the local config , and in the yes or no prompt ask the user if they want to use the account id from local config, if user selects no then we propmpt to provide the account id.

Reply:
We can have a more meaningful name instead of `addAccountID` then I guess.